apps/rehabstod/src/pages/Patient/Patient.tsx

```diff
@@ -33,25 +32,14 @@ export function Patient() {
   const firstCertificate = currentSickness ? currentSickness.intyg[0] : null
   const isDoctor = user?.urval === UserUrval.ISSUED_BY_ME
 
-  const { state } = useLocation()
-
   return (
     <PatientContext.Provider value={patientState}>
       {patient ? <PatientHeader patient={patient} /> : <PatientErrorHeader />}
       <OpenTabsDialog />
       <div className="ids-content m-auto max-w-7xl py-10 px-2.5">
         {!error && (
           <div className="flex justify-end gap-5">
-            {currentSickness && firstCertificate && (
-              <div className="w-64">
-                <SelectRekoStatus
-                  endDate={currentSickness.slut}
-                  patientId={firstCertificate.patient.id}
-                  statusFromSickLeave={state.rekoStatus}
-                  rekoStatusTypes={populatedFilters ? populatedFilters.rekoStatusTypes : []}
-                />
-              </div>
-            )}
+            <PatientRekoStatus currentSickLeaves={currentSickLeaves} earlierSickLeaves={earlierSickLeaves} />
```

Comment:
Cool that u added this component aswell. A minor suggestion that you are free to ignore if u disagree, but since we already have the logic related to the certificates we need, should we use it instead of sending in the whole list? This might affect the reusability of the component, so I let you decide if it makes sense or not.

```javascript
<PatientRekoStatus currentSickLeaves={currentSickness} earlierSickLeaves={firstCertificate} />
```

Reply:
We don't have the logic for the first certificate of the earlierSickLeaves so that's why I placed it in the component to remove some of that logic since it's only used by PatientRekoStatus. The firstCertificate here is the first certificate of the first active sick leave, currentSickness